### PR TITLE
Updated pipeline to use UMCCR AWS accounts

### PR DESCRIPTION
**Pull request by victorskl**
* Dropped make build step as it will be part of cdk synth
* Improved README and typography

### CODE REVIEW

**Review by williamputraintan — APPROVED** [review]

**Comment by victorskl** [comment]
@reisingerf While we are waiting for UoM accounts, we shall start playing with our current ones. It will be also good practice to see tear up/down back to clean slate, etc..

**Comment by victorskl** [comment]
[Orcabus V1: Implement Runtime Lambda Layer building as CDK synth](https://trello.com/c/vc3916yg/1423-orcabus-v1-implement-runtime-lambda-layer-building-as-cdk-synth)

**Comment by victorskl** [comment]
Updating tags in tick

**Review by reisingerf — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -23,10 +23,7 @@ baseline:
 
 test:
 	@yarn test
-	# @pytest
-
-build:
-	@for dir in $(shell find ./lib/workload/stateless/layers -maxdepth 1 -mindepth 1 -type d -exec basename {} \;); do ./lib/workload/stateless/layers/create_layer_package.sh $$dir; done
+	@pytest
```
[the remark this answers is not in the document]
Comment (williamputraintan) [reply Makefile:26]: 🙋, I am the culprit here! I commented out as I was trying on the `make test` (for `yarn test`) and it failed as pytest wasn't configure that properly there. Should have make it clear for the comment or just uncomment it back.

[the remark this answers is not in the document]
Comment (victorskl) [reply Makefile:26]: All good! Early days...

**Pull request merged**